cli/commands/metadata_apply.go

```diff
@@ -19,8 +19,15 @@ func newMetadataApplyCmd(ec *cli.ExecutionContext) *cobra.Command {
 	metadataApplyCmd := &cobra.Command{
 		Use:   "apply",
 		Short: "Apply Hasura metadata on a database",
-		Example: `  # Apply Hasura GraphQL Engine metadata present in metadata.[yaml|json] file:
-  hasura metadata apply`,
+		Example: `
+	# Apply Hasura GraphQL Engine metadata present in metadata.[yaml|json] file:
```

Comment:
Change these to spaces. 1st line should start after the backquote and two spaces.